src/main/scala/firrtl/transforms/ConstantPropagation.scala

```diff
@@ -99,14 +108,22 @@ class ConstantPropagation extends Transform with ResolvedAnnotationPaths {
 
   override val annotationClasses: Traversable[Class[_]] = Seq(classOf[DontTouchAnnotation])
 
-  trait FoldCommutativeOp {
+  sealed trait SimplifyBinaryOp {
```

Comment:
Not that renaming a trait is not binary compatible so this isn't backportable

Reply:
Oh I see, misread the diff, you didn't remove it, but you did seal it which is probably not binary compatible since someone could extend it. I like sealing it though, maybe on the backport PR we undo the sealing?

Reply:
A new virtual method `matchingArgsValue` without a default implementation is definitely not backportable though, so maybe we just hold this improvement off till 1.3

Reply:
It _does_ have a default implementation in the trait that already existed before (`FoldCommutativeOp`).  The weird diff is not helping :)

Reply:
Oh yes, I see it, thanks!